### DISCUSSION TITLE
Project structure of "dagster-open-platform"

### DISCUSSION THREAD

**Discussion by hubmos** [dagster-io/dagster#25779] [Q&A] [answered]
Hi,

It seems that the project structure of dagster-open-platform differs from your own recommendations in the documentation:
https://docs.dagster.io/guides/dagster/recommended-project-structure

In the docs you recommend that we group by assets/resources first, then by business domain/data source, but in your own project you actually just group by data source (?).

Hope you can give some clarification on this, thanks.

**Comment by cmpadden** [comment, marked as answer]
Hi @hubmos -

We are actually changing our  recommendation on project structure, which can be found in the Preview of the new docs site. Though, this is still subject to change as we continue to refine things.

https://docs-preview.dagster.io/guides/project-structure

Our belief is that there are many ways to organize a project depending on the structure and scale of your team, along with personal preference.

If you have any feedback on the new doc above, please let me know! Would love to make sure it makes sense.
